README.md

```diff
@@ -1,47 +1,115 @@
 # Mimir bundle
 
-Mimir is a shardable, horizontally scalable push-based metrics server, and alert rule evaluation engine which is a Prometheus/Cortex/Thanos-compatible by Grafana Labs.
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Mimir bundle](#mimir-bundle)
+    - [Usage](#usage)
+        - [Recommended or minimal deployment](#recommended-or-minimal-deployment)
+        - [Reader-Writer path deployment:](#reader-writer-path-deployment)
+    - [Overlays](#overlays)
+
+<!-- markdown-toc end -->
+
+
+
+Mimir is an open source, horizontally scalable, highly available, multi-tenant TSDB for long-term storage for Prometheus.
 
 This Juju bundle deploys Mimir and a small object storage server, consisting of the following interrelated charms:
 
-- [Mimir](https://charmhub.io/mimir-k8s) ([source](https://github.com/canonical/mimir-k8s-operator))
-- [s3proxy](https://charmhub.io/s3proxy-k8s) ([source](https://github.com/canonical/s3proxy-k8s-operator))
+- [Mimir Coordinator](https://charmhub.io/mimir-coordinator-k8s) ([source](https://github.com/canonical/mimir-coordinator-k8s-operator))
+- [Mimir Worker](https://charmhub.io/mimir-worker-k8s) ([source](https://github.com/canonical/mimir-worker-k8s-operator))
+- [s3integrator](https://charmhub.io/s3-integrator) ([source](https://github.com/canonical/s3-integrator))
 
 This bundle is under development.
-Join us on [Discourse](https://discourse.charmhub.io/t/canonical-observability-stack/5132) and [MatterMost](https://chat.charmhub.io/charmhub/channels/observability)!
+Join us on:
+
+- [Discourse](https://charmhub.io/topics/canonical-observability-stack)
+- [Matrix chat](https://matrix.to/#/#cos:ubuntu.com)
 
 ## Usage
 
-Before deploying the bundle you may want to create a dedicated model for COS components, of which Mimir is one, if one does not already exist:
+Before deploying the bundle you may want to create a dedicated model for Mimir components:
 
 ```shell
-juju add-model cos
-juju switch cos
+juju add-model mimir
+juju switch mimir
 ```
 
-You can deploy the bundle from charmhub with:
+### Recommended or minimal deployment
+
+Deploy the bundle from a local file by running:
 
 ```shell
-juju deploy mimir-bundle --channel=edge --trust
+tox -e render-bundle -- bundle.yaml --template=bundle.yaml.j2 --channel=edge --recommended_deployment=True
+juju deploy ./bundle.yaml --trust
 ```
 
-or, to deploy the bundle from a local file:
+Note that the `--recommended_deployment=True` parameter will generate a bundle with:
```

Comment:
should we call it `HA` instead of `recommended`? or is HA a bit overloaded? WDYT @simskij

Reply:
I would go with `distributed`. The default should likely reflect what is recommended to start with, hence it being a default, while `distributed` would beef it out from the get-go. I agrree that `HA` is too overloaded.

Reply:
From my point of view `HA` can be interpreted in many different ways

Reply:
🆙 I also think this should be called **distributed** instead of recommended :)